### PR TITLE
handle bignumber internally

### PR DESCRIPTION
**Pull request by tarcisiozf**
Because the performance manager uses the `bignumber.js` lib, this dependency end-up spreading across other libraries, it should accept any type of number and convert it internally.

Breaking change:
This PR also changes the method signature of `addOrder` to use the same signature as `canOpenOrder`.

### CODE REVIEW

File: src/PerformanceManager.js [hunk 1]
```diff
@@ -4,17 +4,17 @@ const BigNumber = require('bignumber.js')
...
     maxPositionSize,
     allocation
   }) {
     super()
 
-    this.maxPositionSize = maxPositionSize
-    this.allocation = allocation
+    this.maxPositionSize = maxPositionSize && new BigNumber(maxPositionSize)
+    this.allocation = allocation && new BigNumber(allocation)
     this.availableFunds = new BigNumber(allocation)
```
Comment (avsek477) [inline src/PerformanceManager.js:21]: this.availableFunds = allocation && new BigNumber(allocation || SOME_DEFAULT_VALUE or 0) ? 
Same for `peak` and `trough` as well

**Review by avsek477 — CHANGES_REQUESTED** [review]
rest, lgtm

File: src/PerformanceManager.js (hunk 1 above)
Comment (tarcisiozf) [reply src/PerformanceManager.js:21]: fixed 69241fc61bbde8154c9ac42d537895391f78701c

**Review by avsek477 — APPROVED** [review]
lgtm

**Pull request merged**